src/views/Support/Support.js

```diff
@@ -132,6 +132,29 @@ const useStyles = makeStyles((theme) => ({
 
 let socket = null;
 
+const pageSize = 20;
+
+const buildUserQuery = (userPage, userOffset) => {
```

Comment:
1. should use pageSize as argument of the method.
2.method build be abstract as a common method 'buildQuery' in services/api/ApiService.js